server/services/star.js

```diff
@@ -25,7 +25,7 @@ module.exports = class StarService extends EventEmitter {
             name,
             naturalResources,
             location,
-            infrastructure: { }
+            infrastructure: { a: 'Makes sure it doesnt get deleted'}
```

Comment:
???

Reply:
Yeah the idea is very weird, but if this was an empty object, it would be returned fine the first time, but the second time the full array of stars was returned, each star had this object empty, and it was not found in the data. So since this one gets overwritten anyway later on, this value can be set so it isnt empty and doesnt get removed...

Yeah, I dont understand JS either....